### ISSUE TITLE
VPN causes buffering error

### ISSUE THREAD

**Issue by KyleKatzen** [swar/nba_api#30]
When attempting to pull anything from the api, if I'm using my VPN, specifically Private Internet Access, I get this error. It hangs until I interrupt it. 

```python
from nba_api.stats.endpoints import leaguegamelog
season201819 = leaguegamelog.LeagueGameLog()

TypeError                                 Traceback (most recent call last)
~/anaconda3/lib/python3.6/site-packages/urllib3/connectionpool.py in _make_request(self, conn, method, url, timeout, chunked, **httplib_request_kw)
    379             try:  # Python 2.7, use buffering of HTTP responses
--> 380                 httplib_response = conn.getresponse(buffering=True)
    381             except TypeError:  # Python 2.6 and older, Python 3

TypeError: getresponse() got an unexpected keyword argument 'buffering'

During handling of the above exception, another exception occurred:

KeyboardInterrupt                         Traceback (most recent call last)
```
etc.

The solution was simply to disconnect my VPN, but obviously I'd still like to use it when I'm doing work on public wifi.

**Comment by swar** [comment]
Here's four things I would try:

1.) Run the script as admin.
2.) PIA's pool may be blocked. You could try switching to another server, but check if you can visit the links via browser.
3.) Try switching headers or using a public proxy. The headers might be blocked when accessed through PIA.
4.) if none of these work, then Python might not be routing traffic through the VPN and thus getting blocked by PIA's kill switch. Try opening Python after a connection to PIA has been established.

If none of these work, I have no idea. PIA is working fine for me with proxies.

**Comment by eroma934** [comment]
@KyleKatzen for what it's worth, I was running into a similar issue with both PIA and on GCP. My guess is that the NBA is blocking certain IPs. 

Similarly to @swar, I found that adding a proxy to the debug.py file seems to work. Note that you'll like experience a performance hit if you rely on free proxies. It looks like others have run into similar [issues](https://towardsdatascience.com/pretending-to-know-about-the-nba-using-python-699177a58685).

Ironically, if I use headers and cookies I can run a curl command from behind the VPN and on GCP without any issue, but running the same parameters through requests causes it to hang. Not sure why that is, but I'm not the most versed in how curl handles the responses differently than requests. Hope this helps.

**Comment by swar** [comment]
Feel free to create a new ticket if this issue still persists.
